server/src/Schemas/channels.js

```diff
@@ -12,6 +12,6 @@ const ChannelSchema = new Schema({
   autoJoin: Boolean
 });
 
-const ChannelModel = mongoose.model('ChannelSchema', ChannelSchema);
+const ChannelModel = mongoose.model('ChannelsSchema', ChannelSchema);
```

Comment:
Try to create a constant file for storing all the string constants.

Reply:
fixed